eq.yml

```diff
@@ -955,41 +940,119 @@ jobs:
       TF_VAR_ecs_cluster_name: 'preprod-eq'
       TF_VAR_docker_registry: {{docker_registry}}
       TF_VAR_aws_alb_arn: {{preprod_aws_alb_arn}}
-      TF_VAR_survey_launcher_url: https://preprod-new-surveys-launch.eq.ons.digital
-      TF_VAR_application_cidrs: [10.30.21.192/28,10.30.21.208/28,10.30.21.224/28]
-      TF_VAR_firebase_project_id: ((preprod_author_firebase_project_id))
-      TF_VAR_firebase_api_key: ((preprod_author_firebase_api_key))
-      TF_VAR_firebase_messaging_sender_id: ((preprod_author_firebase_messaging_sender_id))
-      TF_VAR_schema_validator_url: https://preprod-schema-validator.eq.ons.digital
+      TF_VAR_service_name: 'author'
+      TF_VAR_container_name: 'eq-author'
+      TF_VAR_container_port: 3000
+      TF_VAR_listener_rule_priority: 102
+      TF_VAR_healthcheck_path: '/status.json'
+      TF_VAR_healthcheck_grace_period_seconds: 60
+      TF_VAR_slack_alert_sns_arn: '((preprod_slack_alert_sns_arn))'
+      TF_VAR_high_cpu_threshold: 80
     config:
       platform: linux
       image_resource:
         type: docker-image
         source:
           repository: onsdigital/eq-terraform-build
       inputs:
-      - name: eq-author-deploy
       - name: eq-author
-      - name: eq-author-api
-      - name: eq-publisher
+      - name: eq-ecs-deploy
       run:
         path: bash
         args:
         - -exc
         - |
-          cd eq-author-deploy
-          tfenv install 0.10.8
+          cd eq-ecs-deploy
+          tfenv install
 
           author_tag=$(cat ../eq-author/.git/HEAD | xargs echo -n)
+
+          terraform init -backend-config="bucket="concourse-preprod-terraform-state"" -backend-config="key="preprod-author""
+          terraform apply \
+          -var container_tag=$author_tag \
+          -var 'container_environment_variables="{ \"name\": \"REACT_APP_BASE_NAME\", \"value\": \"/eq-author\" }, { \"name\": \"REACT_APP_USE_MOCK_API\", \"value\": \"false\" }, { \"name\": \"REACT_APP_API_URL\", \"value\": \"https://preprod-author-api.eq.ons.digital/graphql\" }, { \"name\": \"REACT_APP_PUBLISHER_URL\", \"value\": \"https://preprod-publisher.eq.ons.digital/publish\" }, { \"name\": \"REACT_APP_GO_LAUNCH_A_SURVEY_URL\", \"value\": \"https://preprod-new-surveys-launch.eq.ons.digital/quick-launch\" }, { \"name\": \"REACT_APP_USE_FULLSTORY\", \"value\": \"true\" }, { \"name\": \"REACT_APP_USE_SENTRY\", \"value\": \"true\" }, { \"name\": \"REACT_APP_ENABLE_AUTH\", \"value\": \"true\" }, { \"name\": \"REACT_APP_FIREBASE_PROJECT_ID\", \"value\": \"((preprod_author_firebase_project_id))\" }, { \"name\": \"REACT_APP_FIREBASE_API_KEY\", \"value\": \"((preprod_author_firebase_api_key))\" }, { \"name\": \"REACT_APP_FIREBASE_MESSAGING_SENDER_ID\", \"value\": \"((preprod_author_firebase_messaging_sender_id))\" }"'
```

Comment:
It might be worth seeing if we can use a multiline string here?

Reply:
I would assume that `REACT_APP_USE_FULLSTORY`, `REACT_APP_USE_SENTRY` and `REACT_APP_ENABLE_AUTH` default to true?